### PR TITLE
VZ-7041: Validate the view-user role for verrazzano user

### PR DESCRIPTION
**Pull request by bhatpmk**
Adds a new test case to validate the verrazzano user has view-user client role. Also updated the debug messages in keycloak.go

### CODE REVIEW

File: tests/e2e/verify-install/keycloak/keycloak_test.go [hunk 1]
```diff
@@ -21,9 +21,13 @@ import (
 )
 
 const (
-	waitTimeout              = 10 * time.Minute
-	pollingInterval          = 30 * time.Second
-	keycloakNamespace string = "keycloak"
+	waitTimeout       = 10 * time.Minute
```
Comment (lake-of-dreams) [inline tests/e2e/verify-install/keycloak/keycloak_test.go:24]: aren't we able to use these constants from keycloak package?

File: tests/e2e/verify-install/keycloak/keycloak_test.go [hunk 2]
```diff
@@ -32,6 +36,12 @@ type KeycloakClients []struct {
 	ClientID string `json:"clientId"`
 }
 
+// KeycloakRoles represents an array of roles configured in Keycloak
+type KeycloakRoles []struct {
```
Comment (lake-of-dreams) [inline tests/e2e/verify-install/keycloak/keycloak_test.go:40]: is it not importable from keycloak package

File: tests/e2e/verify-install/keycloak/keycloak_test.go [hunk 3]
```diff
@@ -100,12 +110,25 @@ var volumeClaims map[string]*corev1.PersistentVolumeClaim
...
 		volumeClaims, err = pkg.GetPersistentVolumeClaims(keycloakNamespace)
 		return volumeClaims, err
 	}, waitTimeout, pollingInterval).ShouldNot(BeNil())
+
+	kubeconfigPath, err := k8sutil.GetKubeConfigLocation()
+	if err != nil {
+		Fail(fmt.Sprintf("Failed to get default kubeconfig path: %s", err.Error()))
+	}
+	isKeycloakEnabled = pkg.IsKeycloakEnabled(kubeconfigPath)
+	isMinVersion140, err = pkg.IsVerrazzanoMinVersion("1.4.0", kubeconfigPath)
```
Comment (lake-of-dreams) [inline tests/e2e/verify-install/keycloak/keycloak_test.go:128]: we are again checking it in line 337 below, should be changed there as well to just use this global value

File: tests/e2e/verify-install/keycloak/keycloak_test.go [hunk 4]
```diff
@@ -500,3 +520,66 @@ func verifyRancherClientURIs(keycloakClient *Client, env string) bool {
...
+func verifyUserClientRole(user, userRole string) bool {
+	var keycloakRoles KeycloakRoles
+
+	// Login to Keycloak
+	if !loginKeycloak() {
+		return false
+	}
+
+	// Get the roles for the verrazzano user
+	cmd := exec.Command("kubectl", "exec", "keycloak-0", "-n", "keycloak", "-c", "keycloak", "--", "/opt/jboss/keycloak/bin/kcadm.sh", "get-roles", "-r", vzSysRealm, "--uusername", user, "--cclientid", realmMgmt, "--effective", "--fields", "id,name")
```
Comment (lake-of-dreams) [inline tests/e2e/verify-install/keycloak/keycloak_test.go:560]: do we really need the id here, cant't it just be `--fields name`

File: tests/e2e/verify-install/keycloak/keycloak_test.go (hunk 2 above)
Comment (bhatpmk) [reply tests/e2e/verify-install/keycloak/keycloak_test.go]: The constants in keycloak package are not exported, prefer to do that later as a follow up change

File: tests/e2e/verify-install/keycloak/keycloak_test.go (hunk 3 above)
Comment (bhatpmk) [reply tests/e2e/verify-install/keycloak/keycloak_test.go:127]: Done

File: tests/e2e/verify-install/keycloak/keycloak_test.go (hunk 4 above)
Comment (bhatpmk) [reply tests/e2e/verify-install/keycloak/keycloak_test.go]: Updated to just get the name

File: tests/e2e/verify-install/keycloak/keycloak_test.go (hunk 1 above)
Comment (bhatpmk) [reply tests/e2e/verify-install/keycloak/keycloak_test.go:24]: The constants in keycloak package are not exported, prefer to do the refactoring later.

**Review by lake-of-dreams — APPROVED** [review]
LGTM

**Pull request merged**